data/companies.yml

```diff
@@ -462,3 +462,6 @@ users:
   - name: "Zendesk"
     logo: "/logos/zendesk.svg"
     url: "https://www.zendesk.com/"
+  - name: "Bitvavo"
+    logo: "/press/black/bitvavo-mark-and-logo/bitvavo-mark-and-logo-black.svg"
```

Comment:
The .svg file should be added in the `static/logos` directory, and then the line here would be /logos/{svg file}.

This will fix the lint error as well (it can't find the logo file).